extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/simulate/KafkaResource.java

```diff
@@ -128,6 +128,17 @@ public void produceRecordsToTopic(List<ProducerRecord<byte[], byte[]>> records)
     }
   }
 
+  /**
+   * Produces records to a topic of this embedded Kafka server.
+   */
+  public void produceRecordsToTopic(List<ProducerRecord<byte[], byte[]>> records)
+  {
+    produceRecordsToTopicWithExtraProperties(
+        records,
+        null
+    );
```

Comment:
Nit: can probably be in a single line
```suggestion
    produceRecordsToTopicWithExtraProperties(records, null);
```